### ISSUE TITLE
Hashtags become hex color chips

### ISSUE THREAD

**Issue by Breton** [tootsuite/mastodon#14059]
### Pitch

Imagine: Hashtags that follow the pattern of hex colours, such as #FFAA00 or #CCC become conversation channels for fashion, design or general conversation about that colour, similar to dribbble, or pinterest search by colour features. No new feature need be implemented for that! However, what if a hex colour automatically rendered as a colour swatch for the corresponding colour, similar to a hashflag. Tapping the swatch opens posts related to that colour swatch. 

### Motivation

People do occasionally post hex colours on mastodon, and they do already (one could argue falsely?) get parsed as though they are hashtags. If the software is going to do anything with these, wouldn't it be nice to see what the colours actually are?

**Comment by ariasuni** [comment]
One problem I see is that some valid colors can also be legit hashtags. You use CCC as an example: it can have [a lot of meanings](https://en.wikipedia.org/wiki/CCC). Same could be said for Ada. It seems it’s not really a problem for 6-digits colors though.

**Comment by Breton** [comment]
good point! in principle though, is there harm in putting a light gray circle next to #CCC or a magenta circle next to #ADA ? maybe it’s a little confusing but it doesn’t stop the hashtag from having non color meanings. nevertheless, you have a point.

**Comment by Breton** [comment]
I just found this: Hashtags in this zettler program can be assigned colors by the user.
https://docs.zettlr.com/en/reference/settings/#manage-tags
